libs/sdk-bindings/bindings-android/lib/src/main/kotlin/breez_sdk_notification/job/LnurlPayInvoice.kt

```diff
@@ -41,7 +42,12 @@ class LnurlPayInvoiceJob(
         try {
             request = Json.decodeFromString<LnurlInvoiceRequest>(LnurlInvoiceRequest.serializer(), payload)
             val nodeState = breezSDK.nodeInfo()
-            if (request.amount < 1000UL || request.amount > nodeState.inboundLiquidityMsats) {
+            val maxSendableMsats: ULong =
+                maxOf(
+                    ExtensionSettings.instance.autoChannelSetupFeeLimitMsats,
```

Comment:
Using a global singleton here is problematic. It makes the getServiceSettings abstraction redundant.
Also IMO it makes the input flow to the jobs complicated. A unified flow where all the job input is defined and passed in one place is better, this is why I think passing it as an argument on the intent makes sense.